driver/bpf/filler_helpers.h

```diff
@@ -25,6 +25,15 @@ or GPL2.txt for full copies of the license.
 #define MAX_PATH_COMPONENTS 16
 #define MAX_PATH_LENGTH 4096
 
+/* This enum is used to tell our helpers if they have to
+ * read from kernel or user memory.
+ */
+enum read_memory
+{
+	USER = 0,
+	KERNEL = 1,
```

Comment:
This is an ultra-nitpick, but after seeing these enums in the code it is a little cryptic what `USER` and `KERNEL` might refer to, maybe we can rename them to something like `USER_MEMORY` and `KERNEL_MEMORY`, WDYT?

Reply:
I actually simply re-used `enum read_memory` from the modern bpf driver. Let me check try to find a place to have it common to both bpf drivers. We can then have a follow-up change for all places in bpf and moden bpf driver.